package.json

```
@@ -7,6 +7,7 @@
   },
   "scripts": {
     "build": "cross-env NODE_ENV=production FORCE_COLOR=1 lerna run build --stream",
+    "build:dev": "FORCE_COLOR=1 lerna run build --stream",
```

this is missing cross-env
```suggestion
    "build:dev": "cross-env FORCE_COLOR=1 lerna run build --stream",
```

---

Updated!